src/TraceParserGen/TraceParserGen.cs

```diff
@@ -945,20 +926,16 @@ private string GetType(List<FieldInfo> versions)
     /// </summary>
     public static string ToCSharpName(string input)
     {
-        if (input == null)
-        {
-            return null;
-        }
+        if (String.IsNullOrEmpty(input))
```

Comment:
💡 

```suggestion
        if (string.IsNullOrEmpty(input))
```

Reply:
Again - the existing code is mixed, but I'll gladly adhere to this convention.